src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,29 @@
+.catalog {
+  --column-count: 1;
+
+  display: grid;
+  box-sizing: border-box;
+  padding: 50px 40px;
+  grid-template-columns: repeat(auto-fill, 200px);
+  grid-template-rows: repeat(2, 1fr);
+  gap: 46px 48px;
+  place-content: center;
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .catalog {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .catalog {
+    --column-count: 4;
+  }
+}
```

Comment:
In these media queries, you're changing the `--column-count` variable, but it's not used anywhere in your CSS. If you intended to use this variable to control the number of columns in your grid layout, make sure to use it in the `grid-template-columns` property.

Reply:
<img width="623" alt="image" src="https://github.com/user-attachments/assets/099a0cec-09f8-43d6-80fb-51dd53ab6269">
this comment makes sense